var/spack/repos/builtin/packages/octave/package.py

```diff
@@ -268,6 +268,12 @@ def configure_args(self):
             ])
         else:
             config_args.append("--without-z")
+            
+        # If 64-bit BLAS is used:
+        if (spec.satisfies('^openblas+ilp64') or
+            spec.satisfies('^intel-mkl+ilp64') or
+            spec.satisfies('^intel-parallel-studio+mkl+ilp64')):
+            config_args.append('F77_INTEGER_8_FLAG=-fdefault-integer-8')
```

Comment:
My only concern is which compilers actually support this flag.

Reply:
@adamjstewart This is no compiler flag. It is an Octave "configure" flag / argument in spack (config_args).